docs/framework/github_feedback.md

```diff
@@ -0,0 +1,175 @@
+
+### Footer feedback
+
+Next, open the interview that you want to add feedback to in docassemble's playground.
+
+If you include `assembly_line.yml` in your interview already, all you need to do is add the following block to set the `al_github_repo` variable, which `assembly_line.yml` uses to setup the feedback interview correctly.
+
+```yml
+code: |
+  github_repo_name = "YOUR_REPO_NAME" # i.e. "docassemble-MyInterview"
+```
+
+:::tip Duplicate blocks
+
+Use [docassemble's playground search](https://docassemble.org/docs/playground.html#interview_files) to make sure that you only have one block defining the `github_repo_name` variable. If you have more than one, only the last block will be used, which can be confusing.
+:::
+
+If you aren't using `assembly_line.yml`, you can
+add the below two blocks to add a footer to every page of your interview:
+
+```yml
+---
+modules:
+  - docassemble.GithubFeedbackForm.github_issue
+---
+default screen parts:
+  footer: |
+    [:comment-dots: Feedback](${ feedback_link(user_info(), github_repo=YOUR_REPO_NAME) } ){:target="_blank"}
+```
+
+You can then start your interview, and select the Feedback link in the footer. Fill in the feedback interview, and submit the feedback. If
+you setup GitHub integration, there should be a link to the newly-created GitHub issue. Otherwise, you can view the feedback [using the feedback browsing interview](#how-to-browse-feedback-on-the-server).
+
+
+## How to get reviews in an interview
+
+While having a feedback link in the footer is useful for users who are specifically seeking to give feedback, you can take a more proactive approach and let users
+give quick feedback with an in-interview review widget. The final product of this guide will look like the below screenshot:
+
+![A screenshot of the review widget. The widget is outlined in a light gray box has the text "Did we help you?", with two buttons below, one with a thumbs up and one with a thumbs down](../assets/github_feedback_review_widget.png)
+
+First, find an interview you'd like to add a feedback widget too. Where exactly you want to ask for feedback is up to you, but we'd recommend doing it at the end of the interview, when someone has reached an end point.
+
+For this how-to, we'll make an short example interview from scratch. Then entire interview is below:
+
+```yml
+include:
+  - docassemble.GithubFeedbackForm:embedded_review.yml
+---
+mandatory: True
+question: |
+  First screen
+subquestion: |
+  Welcome to this example interview.
+
+  ${ only_once_review_widget() }
+```
+
+The key lines in the above are:
+
+* adding `docassembl.GithubFeedbackForm:embedded_review.yml` to the `include` block, and
+* `${ only_once_review_widget() }` in the `subquestion` attribute of a question block
+
+If you are adding the review widget to an existing interview, focus on just those two lines.
```

Comment:
Trying to avoid people focusing on

```yaml
mandatory: True
question: |
  First screen
subquestion: |
  Welcome to this example interview.
```

The other case isn't adding it to a new interview, it's just following along with this example interview. I would just say "don't add this to a new interview; wait until you have an MVP interview before worrying about the feedback widget". Not sure if that's worth adding here though.

Reply:
I think it might actually be less confusing to just delete the line in that case. I don't think the clarification about the question itself is necessary and I think when to add it doesn't matter as much. It doesn't really matter when they add it. It won't really do anything till they make it live. Sometimes I add stuff when I'm thinking about it (so I won't forget), not when it would be logical in a timeline.

The below will leave an extra blank line in the docs unfortunately.

```suggestion
```

Reply:
I changed it manually, so no need to worry about the extra line.